docs-gen/content/docs/getting-started/overview.md

```diff
@@ -3,54 +3,23 @@ title: "Overview"
 weight: 1
 ---
 
-The Kubeflow Pipelines Operator provides a declarative API for managing and running ML pipelines with Resource Definitions on multiple providers.
+The Kubeflow Pipelines Operator (KFP Operator) provides a declarative API for managing and running ML pipelines with Resource Definitions on multiple providers.
 A provider is a runtime environment for managing and executing ML pipelines and related resources.
 
-## Compatibility
+### Why KFP Operator
 
-The operator currently supports
-- TFX Pipelines with Python 3.7 and 3.9 - pipelines created using the KFP DSL are not supported yet
-- KFP standalone (a full KFP installation is not supported yet) and Vertex AI
+The idea behind the project came about because as engineers we wanted to apply our engineering best practices behind the 
+process of deploying a trained model to be served in a production environment. We wanted to get away from the mentality of 
+"I just need this trained model in Production" or the "If it's 10 lines of code I'd rather copy and paste it".
```

Comment:
Rather than the quotes, how about something like
```suggestion
We started this project to promote the best engineering practices in the Machine Learning process, while reducing the operational overhead associated with deploying, running and maintaining training pipelines. We wanted to move away from a manual, opaque, copy-and-paste style deployment and closer to a declarative, traceable, and self-serve approach.

```